README.md

```diff
@@ -29,11 +29,20 @@ No need.
 Adding `setup_requires=['setuptools-git-versioning']` somewhere in `setup.py` will automatically download the latest version from PyPi and save it in the `.eggs` folder when `setup.py` is run.
 
 ## Usage
+### pyproject.toml
+Just add this line into your `pyproject.toml`:
 
+```toml
+[tool.setuptools_git_versioning]
+```
+
+### setup.py
 Just add these lines into your `setup.py`:
 
 ```python
-setuptools.setup(version_config=True, setup_requires=["setuptools-git-versioning"], ...)
+setuptools.setup(
+    setuptools_git_versioning=True, setup_requires=["setuptools-git-versioning"], ...
```

Comment:
Didn't get, does it mean that user cannot pass `version_config` to `setup.py` anymore?

Reply:
version_config should continue to work. See the addition to `setup.py` I added the here documented "setuptools_git_versioning" key as an addition to what was already working. This allows the user to use a key that directly relates to the library name. Or they can just stick to version_config.

I can remove this again. I just felt like it would make more sense for the library to offer a key that reflects its own name.

Reply:
I prefer not to add new keys to package configuration if it is possible. Maybe I'll rename existing one in future releases, but not now